airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2301,6 +2325,25 @@ paths:
                 $ref: "#/components/schemas/ImportRead"
         "404":
           $ref: "#/components/responses/NotFoundResponse"
+  /v1/internal/set_temporal_workflow_in_attempt:
+    post:
+      tags:
+        - internal
```

Comment:
I don't see an internal tag at the top of the file. Can we add that and a description clarifying what this means i.e. routes that are for Airbyte operation and not meant to be used externally.

Reply:
@xiaohansong fyi - I'm tagging you in review comments related to your contributions around the internal API, do you have bandwidth to address the related review comments on this branch? Feel free to commit straight to this branch if you don't mind!

Reply:
Let's continue any discussion/changes on this child PR here: https://github.com/airbytehq/airbyte/pull/15902/files